src/org/opendatakit/briefcase/util/ExportToCsv.java

```diff
@@ -74,42 +74,37 @@
 
   private static final Log log = LogFactory.getLog(ExportToCsv.class);
 
-  File outputDir;
-  File outputMediaDir;
-  String baseFilename;
-  BriefcaseFormDefinition briefcaseLfd;
-  TerminationFuture terminationFuture;
-  Map<TreeElement, OutputStreamWriter> fileMap = new HashMap<TreeElement, OutputStreamWriter>();
-  Map<String, String> fileHashMap = new HashMap<String, String>();
-
-  boolean exportMedia = true;
-  Date startDate;
-  Date endDate;
-  boolean overwrite = false;
   int totalFilesSkipped = 0;
   int totalInstances = 0;
   int processedInstances = 0;
-
-
-  public ExportToCsv(File outputDir, BriefcaseFormDefinition lfd, TerminationFuture terminationFuture, Date start, Date end) {
-    this(outputDir, lfd, terminationFuture, lfd.getFormName(), true, false, start, end);
-  }
-
-  public ExportToCsv(File outputDir, BriefcaseFormDefinition lfd, TerminationFuture terminationFuture, String filename, boolean exportMedia, Boolean overwrite, Date start, Date end) {
+  private final Map<TreeElement, OutputStreamWriter> fileMap = new HashMap<TreeElement, OutputStreamWriter>();
+  private final Map<String, String> fileHashMap = new HashMap<String, String>();
+  private final TerminationFuture terminationFuture;
+  private final File outputDir;
+  private final BriefcaseFormDefinition briefcaseLfd;
+  private final String baseFilename;
+  private final boolean exportMedia; // true by default?
+  private final boolean overwrite; // false by default?
+  private final Date startDate;
+  private final Date endDate;
+  private final File outputMediaDir;
+
+  ExportToCsv(TerminationFuture terminationFuture, File outputDir, BriefcaseFormDefinition briefcaseLfd, String baseFilename, boolean exportMedia, boolean overwrite, Date startDate, Date endDate) {
+    this.terminationFuture = terminationFuture;
     this.outputDir = outputDir;
+    this.briefcaseLfd = briefcaseLfd;
+    this.baseFilename = stripCSVExtension(baseFilename);
+    this.exportMedia = exportMedia;
+    this.overwrite = overwrite;
+    this.startDate = startDate;
+    this.endDate = endDate;
     this.outputMediaDir = new File(outputDir, MEDIA_DIR);
-    this.briefcaseLfd = lfd;
-    this.terminationFuture = terminationFuture;
+  }
+
 
+  private String stripCSVExtension(String filename) {
```

Comment:
Remember Josh Bloch’s admonition (do you need the audio file?—:grinning:) stripCsvExtension